app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -714,14 +714,21 @@ def parse_base_item(item)
     end
 
     def parse_image_name(image, image_ref)
+      # parsing using same logic as in docker
+      # https://github.com/docker/docker/blob/348f6529b71502b561aa493e250fd5be248da0d5/reference/reference.go
       parts = %r{
         \A
-          (?:(?:(?<host>[^\.:\/]+\.[^\.:\/]+)|(?:(?<host2>[^:\/]+)(?::(?<port>\d+))))\/)?
+          (?:(?:
+              (?<host>[^\.:\/]+\.[^\.:\/]+)|
+              (?:(?<host2>[^:\/]+)(?::(?<port>\d+)))|
+              (?<localhost>localhost)
+          )\/)?
```

Comment:
Is there any reason we can't have the external group be the capturing one? 
Then we could drop the host|| host2 || localhost logic.